pkg/agent/openflow/pipeline.go

```diff
@@ -1600,6 +1644,19 @@ func (f *featureNetworkPolicy) allowRulesMetricFlows(conjunctionID uint32, ingre
 			Done()
 	}
 	var flows []binding.Flow
+	// For IGMP ingress or multicast egress traffic, we need to calculate the packets dropped or allows by each rule.
```

Comment:
or allowed

Since IGMP and multicast traffic uses a different pipeline

Reply:
done